src/components/Footer/Footer.js

```diff
@@ -44,6 +44,13 @@ const Footer = () => {
         <ActionLink onClick={() => dispatch(setDialogVisible('Impressum'))}>
           {t('Impressum')}
         </ActionLink>
+        <a
+          href="https://doc.trafimage.ch"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          {t('Developer Portal')}
```

Comment:
Is there already translations?

Reply:
Not yet, it will probably stay so on every language, I remove the 't'